plasmapy/utils/decorators/converter.py

```diff
@@ -0,0 +1,38 @@
+"""
+Decorator to convert units of functions in /physics methods
+"""
+from astropy import units as u
```

Comment:
For ease of reading, I generally like to group my imports by `import ...` and `from ... import ...`.  Then I alphabetize them.  This is not a PEP8 thing, just a me thing I think improve readability. :)